### PR TITLE
Browse streams

### PR DESCRIPTION
**Pull request by josh-velasquez**

### CODE REVIEW

File: macosLaunch.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+echo "Launching api rtmpserver and client terminals..."
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+osascript -e 'tell app "Terminal" to do script "cd '$DIR'/api/ && npm start"'
+
+osascript -e 'tell app "Terminal" to do script "cd '$DIR'/rtmpserver/ && npm start"'
+
+osascript -e 'tell app "Terminal" to do script "cd '$DIR'/client/ && npm start"'
```
Comment (emilio-alvarez) [inline macosLaunch.sh]: add a new line to the end of the file pls

Comment (josh-velasquez) [reply macosLaunch.sh]: Update: Added new line at end of file

File: client/src/apis/streams.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import axios from "axios";
 
 export default axios.create({
-  baseURL: "http://localhost:3001"
+  baseURL: "http://192.168.1.78:3001"
```
Comment (emilio-alvarez) [inline client/src/apis/streams.js]: Was this intended to be pushed?

File: client/src/components/streams/StreamShow.js [hunk 3]
```diff
@@ -29,7 +29,7 @@ class StreamShow extends React.Component {
     const { id } = this.props.match.params;
     this.player = flv.createPlayer({
       type: "flv",
-      url: `http://localhost:8000/live/${id}.flv`
+      url: `http://192.168.1.78:8000/live/${id}.flv`
```
Comment (emilio-alvarez) [inline client/src/components/streams/StreamShow.js:32]: Oh it's in this one too.

**Pull request closed without merging**